vmdb/app/apis/vmdbws_ops.rb

```diff
@@ -853,7 +853,7 @@ def getKlassByTag(klass, tag)
 
     $log.info "#{log_header}: enter"
     t0 = Time.now
-    ret = klass.find_tagged_with(:all => tag, :ns => '/managed').all
+    ret = klass.find_tagged_with(:all => tag, :ns => '/managed')
```

Comment:
Was the caller expecting a relation to be returned here?

`MiqProductFeature.select(:id).to_a.delete_if {|f| f.id == 1}` is much different than 
`MiqProductFeature.select(:id).delete_if {|f| f.id == 1}`  # no method error

Some things are even worse, like `delete` which is much different on a relation (deletes from the database) vs. an array.